docs/data/material/getting-started/overview/overview.md

```diff
@@ -0,0 +1,26 @@
+# Material UI - Overview
+
+<p class="description">Material UI is library of React UI components that implements Google's Material Design.</p>
+
+## Introduction
+
+Material UI is an open-source React component library that implements Google's [Material Design](https://material.io/).
+
+It includes a comprehensive collection of prebuilt components that are ready for use in production right out of the box.
+
+Material UI is beautiful by design, and features a suite of customization options that make it easy to implement your own custom design system on top of our components.
```

Comment:
Should we mention and link to Base here? Like "For complete unstyled components check out [MUI Base](/base/getting-started/overview/)."